dds_cli/data_remover.py

```diff
@@ -194,3 +194,12 @@ def remove_folder(self, folder):
             raise SystemExit from err
 
         self.__create_failed_table(resp_json=resp_json, level="Folder")
+
+        if resp_json["nr_deleted"]:
```

Comment:
I get:
```
root@3e6a935ee76a:/code# dds --token-path unituser_token data rm -p someunit00002 --folder tests/
     ︵ 
 ︵ (  )   ︵ 
(  ) ) (  (  )   SciLifeLab Data Delivery System 
 ︶  (  ) ) (    http://dds_backend:5000/ 
      ︶ (  )    Version 0.0.12 
          ︶
Current user: unituser_1

Traceback (most recent call last):
  File "/usr/local/bin/dds", line 33, in <module>
    sys.exit(load_entry_point('dds-cli', 'console_scripts', 'dds')())
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/rich_click/rich_click.py", line 594, in main
    return super().main(*args, standalone_mode=False, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/decorators.py", line 38, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/code/dds_cli/__main__.py", line 1397, in rm_data
    remover.remove_folder(folder=folder)
  File "/code/dds_cli/custom_decorators.py", line 176, in create_and_remove_task
    func(self, *args, **kwargs)
  File "/code/dds_cli/data_remover.py", line 198, in remove_folder
    if resp_json["nr_deleted"]:
KeyError: 'nr_deleted'
```

Reply:
Sounds like the backend is not running the matching branch. Should still be fixed.

Reply:
Ok I'll give it a try again and check 👍🏻

Reply:
Yes now there's no problem 👍🏻  My mistake